pkgs/tools/networking/nordvpn/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, lib, fetchurl, openvpn, libxml2, autoPatchelfHook, dpkg }:
+
+stdenv.mkDerivation rec {
+  pname = "nordvpn";
+  version = "3.10.0-1";
+
+  src = fetchurl {
+    url = "https://repo.nordvpn.com/deb/nordvpn/debian/pool/main/nordvpn_${version}_amd64.deb";
+    sha256 = "BNAInjJlQsYpxfUKI13oK/P6n6gpBlvgSQoJAuZ3C2M=";
+  };
+
+  nativeBuildInputs = [ libxml2 autoPatchelfHook dpkg ];
+
+  unpackPhase = ''
+    dpkg -x $src unpacked
+  '';
+
+  installPhase = ''
+    mkdir -p $out/
+    sed -i 's;ExecStart=.*;;g' unpacked/usr/lib/systemd/system/nordvpnd.service
+    cp -r unpacked/* $out/
+    mv $out/usr/* $out/
+    mv $out/sbin/nordvpnd $out/bin/
+    rm -r $out/sbin
+    rm $out/var/lib/nordvpn/openvpn
+    ln -s ${openvpn}/bin/openvpn $out/var/lib/nordvpn/openvpn
```

Comment:
Not sure about this tbh. I suspect they use patched `openvpn` version to provide obfuscation (and probably some other features), because when I try to run obfuscated connection, I see this:

`Jan 16 19:07:28 nixos nordvpnd[1385]: 2022/01/16 19:07:28 [INFO] Options error: Unrecognized option or missing or extra parameter(s) in /var/lib/nordvpn/data/.config.ovpn:21: scramble (2.5.2)`

![image](https://user-images.githubusercontent.com/2827204/149670406-cf9ffdaf-f2e3-4841-9432-f293eb17ae83.png)

FYI: https://tunnelblick.net/cOpenvpn_xorpatch.html

Reply:
@juliosueiras  looks like it is :thinking: 
I removed these 2 rows to keep original `openvpn`
Also, I created a link `/sbin/ip` because `openvpn` calls it
And it works :tada: